core/src/main/java/com/oath/oak/NativeAllocator/OakNativeMemoryAllocator.java

```diff
@@ -41,7 +41,7 @@
     // number of bytes allocated for this Oak among different Blocks
     // can be calculated, but kept for easy access
     private final AtomicLong allocated = new AtomicLong(0);
-
+    private final AtomicLong freeCounter = new AtomicLong(0);
```

Comment:
please add documentation - what does it count? number of bytes?

Reply:
Its just to make objects in free list unique.